src/constants/storage.js

```diff
@@ -0,0 +1,14 @@
+export const STORAGE_PROMETHEUS_QUERIES = {
+  CEPH_STATUS_QUERY: 'ceph_health_status',
+
+  CEPH_OSD_UP_QUERY: 'sum(ceph_osd_up)',
+  CEPH_OSD_DOWN_QUERY: 'count(ceph_osd_up == 0.0) OR vector(0)',
+
+  CAPACITY_STORAGE_TOTAL_BASE_CEPH_METRIC: 'ceph_cluster_total_bytes', // available with Ceph only
```

Comment:
I think it makes more sense to make these queries more generic. I.e. without CAPACITY/UTILIZATION prefixes.

Reply:
Since the names are very convoluted, I'd suggest merging the two PRs that extract those values into constants before renaming them. This should simplify code review, and prevent bugs.

Reply:
ok then